applications/googlehome-meets-dotnetcontainers/GoogleHomeAspNetCoreDemoServer/Dialogflow/Intents/PlatformDescribeHandler.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Copyright(c) 2018 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+using Google.Api.Gax;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace GoogleHomeAspNetCoreDemoServer.Dialogflow.Intents
+{
+    /// <summary>
+    /// Handler for "platform.describe" DialogFlow intent.
+    /// </summary>
+    [Intent("platform.describe")]
+    public class PlatformDescribeHandler : BaseHandler
+    {
+        /// <summary>
+        /// Initializes class with the conversation.
+        /// </summary>
+        /// <param name="conversation"></param>
+        public PlatformDescribeHandler(Conversation conversation) : base(conversation)
+        {
+        }
+
+        /// <summary>
+        /// Handle the intent.
+        /// </summary>
+        /// <param name="req">Conversation request</param>
+        /// <returns></returns>
+        public override async Task<object> Handle(ConvRequest req)
+        {
+            var platform = await Platform.InstanceAsync();
+
+            GetDetailedDescription(platform, out string spokenDescription, out string[] textDescription);
+
```

Comment:
This is more blank lines than I'm used to looking at.

Reply:
Fixed